tests/test_native_ops.py

```diff
@@ -188,3 +188,54 @@ def test_apply_amplitude_damp_single_wire(
         simulator_device_1_wire.apply([ops.AmplitudeDamp(*par, wires=[0])])
 
         assert np.allclose(simulator_device_1_wire.state, expected_density_matrix, **tol)
+
+    @pytest.mark.parametrize("input",
+        [
+            [1, 0, 0, 0],
+            [0, 1, 0, 0],
+            [0, 0, 1, 0],
+            [0, 0, 0, 1],
+        ]
+    )
+    def test_apply_iswap(self, tol, input, shots, analytic):
+        """Tests that applying the CPhase gate yields the expected output."""
```

Comment:
In both of these new tests, the docstring and test name do not match

Reply:
:facepalm: I shouldn't have attempted fixing this so late yesterday night.